src/query/service/src/pipelines/pipeline_builder.rs

```diff
@@ -345,7 +345,10 @@ impl PipelineBuilder {
             &aggregate.agg_funcs,
         )?;
 
-        if self.ctx.get_cluster().is_empty()
+        // this has bugs now, so we disable it for now, cc @winter
```

Comment:
File a new open issue for it?

Reply:
https://github.com/datafuselabs/databend/issues/9673